packages/components/NavBar.tsx

```diff
@@ -14,8 +14,9 @@ import {
 } from '@chakra-ui/react';
 import { HamburgerIcon, CloseIcon } from '@chakra-ui/icons';
 
+import { GithubIcon, LinkedInIcon } from '@packages/components/icons';
+import { Logo } from '@packages/components/icons/Logo';
```

Comment:
Esses dois aqui deveria ser um só né

Reply:
uai tava junto. deu ruim aq pera

Reply:
```suggestion
import { GithubIcon, LinkedInIcon, Logo } from '@packages/components/icons';
```
isso deveria funfar

Reply:
acho que foi my bad mesmo, na hora do fix eu confiei no auto import atualizado e provavelmente nao conferi que ele colocou separado

Reply:
> isso deveria funfar

sim, ta desse jeito aqui. ele foi lintado separado no fix (n sei pq) e eu que deixei passar

Reply:
to subindo dnv ja